content/SCALE/SCALETutorials/Credentials/DirectoryServices/ConfigADSCALE.md

```diff
@@ -23,34 +25,46 @@ Joining an AD domain also configures the Privileged Access Manager (PAM) to let
 
 Users can configure AD services on Windows or Unix-like operating systems using [Samba version 4](https://wiki.samba.org/index.php/Setting_up_Samba_as_an_Active_Directory_Domain_Controller#Provisioning_a_Samba_Active_Directory).
 
-To configure an AD connection, you must know the AD controller domain and the AD system account credentials.
+### Preparing to Configure AD in TrueNAS
+Before configuring Active Directory (AD) in TrueNAs:
 
-### Preparing to Configure AD in SCALE
+you must know the AD controller domain  name and the AD administration account credentials.
+You also need to know the hostname assigned to the TrueNAS system.
 
-Users can take a few steps before configuring Active Directory (AD) to ensure the connection process goes smoothly.
+* Obtain the AD admin account name and password.  
 
-1. Obtain the AD admin account name and password.  
+* [Verify name resolution](#verifying-name-resolution).
+  Go to **Network > Global Network Settings** to verify your TrueNAS network DNS name servers are configured to access the target domain controller you plan to add on the **Active Directory** screen.
 
-3. [Verify name resolution](#verifying-name-resolution)
+* Change the default hostname of the system from **truenas** to the name assigned to the TrueNAS system.
 
-5. [Set time synchronization](#setting-time-synchronization)
+* [Set time synchronization](#setting-time-synchronization)
 
 After taking these actions, you can [connect to the Active Directory domain](#connecting-to-the-active-directory-domain).
 
 ### Verifying Name Resolution
-
 <!-- Revisit **Shell** guidance as part of rework task -->
+{{< hint type="info" >}}
+The TrueNAS administrator account must have access to the Linux shell to perform this function.
+
+Go to **Credentials > Users**, locate the administrator account, click **Edit** and scroll down to the **Shell** setting.
+If not set to a shell option, select **TrueNAS Console**, and click **Save**.
+This opens the TrueNAS **Shell** screen showing the Console Setup menu. 
+
+Return to the **Shell** screen and enter the option to open the Linux shell and press <kbd>Enter</kbd>.
+{{< /hint >}}
 
-To confirm that name resolution is functioning, you can use the **Shell** and issue a `ping` command and a command to check network SRV records and verify DNS resolution.
+To confirm that name resolution is functioning, open the TrueNAS **Shell** and issue a `ping` command and then a command to check network SRV records and verify DNS resolution.
```

Comment:
We perform DNS checks during domain join, but if we're going to provide examples they should include dig commands that explicitly look up relevant AD SRV records. Simple ping is misleading.